tests/TestBaseClass.php

```diff
@@ -161,6 +161,8 @@ public static function installAndActivatePlugin($pluginName)
             $plugin->active = 1;
             $plugin->save();
         }
+
+        return $plugin;
```

Comment:
I wish I could merge just simple fixes like this, so that the size of the PR gets reduced a bit :laughing: